.github/workflows/main.yaml

```diff
@@ -271,3 +286,56 @@ jobs:
       vaultAddr: ${{ inputs.vaultAddr }}
       artifactoryRoleSuffix: ${{ inputs.artifactoryRoleSuffix }}
       slackChannel: ${{ inputs.slackChannel }}
+
+  datadog:
+    name: Push results to datadog
+    runs-on: ubuntu-latest
+    needs:
+      - release
+      - mavenCentral
+      - javadocPublication
+      - testPypi
+      - pypi
+    if: ${{ always() && inputs.pushToDatadog && inputs.dryRun != true }}
+    steps:
+      # Clone gh-action_release repository to run actions locally
+      - uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+        with:
+          # Hardcode the repository name, because GitHub Actions does not provide a built-in context or variable
+          # to directly reference the repository where the reusable workflow is defined
+          repository: SonarSource/gh-action_release
+          # This property is changed during the release process to reference the correct tag
+          # During development change this to your branch name to run it in another repository
+          ref: feat/jd/BUILD-5228-releasability-outputs
```

Comment:
I will have to merge this PR first and then make another PR to change this ref to ` ${{ github.ref }}` 
Same for the checkout step at the top